mflowgen/Tile_MemCore/construct.py

```diff
@@ -150,8 +162,6 @@ def construct():
   place.extend_inputs( ["sdc"] )
   cts.extend_inputs( ["sdc"] )
 
-  xcelium_sim.extend_inputs( ["array_rtl.v", "CW_fp_mult.v", "CW_fp_add.v", "sram.v"] )
```

Comment:
Do we not need this anymore?

Reply:
The `xcelium_sim` step was moved into the testbench step.